engines/wintermute/detection.cpp

```diff
@@ -210,6 +214,1425 @@ class WintermuteMetaEngine : public AdvancedMetaEngine {
 		pm.getSaveStateDesc(slot, retVal);
 		return retVal;
 	}
+
+	Common::KeymapArray initKeymaps(const char *target) const {
+		using namespace Common;
+
+		Keymap *engineKeyMap = new Keymap(Keymap::kKeymapTypeGame, "wintermute", "Wintermute engine");
+
+		Action *act;
+		
+		act = new Action("LCLK", _("Left Click"));
+		act->setLeftClickEvent();
+		act->addDefaultInputMapping("MOUSE_LEFT"); // original mouse
+		act->addDefaultInputMapping("JOY_A"); // extra joy
+		engineKeyMap->addAction(act);
+
+		act = new Action("RCLK", _("Right Click"));
+		act->setRightClickEvent();
+		act->addDefaultInputMapping("MOUSE_RIGHT"); // original mouse
+		act->addDefaultInputMapping("JOY_B"); // extra joy
+		engineKeyMap->addAction(act);
+
+		act = new Action("RETURN", _("Confirm"));
+		act->setKeyEvent(KeyState(KEYCODE_RETURN, ASCII_RETURN));
+		act->addDefaultInputMapping("RETURN"); // original keyboard
+//TODO: extra joy control, e.g. "JOY_R+JOY_X"
```

Comment:
Please, indent all of these TODO comments properly (more follow below, no reason to repeat this comment)

Reply:
Intended in line with code with tabs.